lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -538,12 +538,12 @@ def test_build_student_data_for_block_without_generate_report_data(self, mock_li
             student_data, _ = ProblemResponses._build_student_data(
                 user_id=self.instructor.id,
                 course_key=self.course.id,
-                usage_key_str=str(problem.location),
+                usage_key_str_list=[str(problem.location)],
```

Comment:
`usage_key_str_list ` is not used in ironwood release (ref: https://github.com/edx/edx-platform/blob/441d6384eef5ed0b62e936a7d85958d10e43d92e/lms/djangoapps/instructor_task/tests/test_tasks_helper.py#L541)